packages/jest-core/src/SearchSource.ts

```diff
@@ -322,4 +331,29 @@ export default class SearchSource {
 
     return searchResult;
   }
+
+  findRelatedSourcesFromTestsInChangedFiles(
+    changedFilesInfo: ChangedFiles,
+  ): Array<string> {
+    if (!hasSCM(changedFilesInfo)) {
+      return [];
+    }
+    const {changedFiles} = changedFilesInfo;
+    const dependencyResolver = new DependencyResolver(
+      this._context.resolver,
+      this._context.hasteFS,
+      buildSnapshotResolver(this._context.config),
+    );
+    const relatedSourcesSet = new Set<string>();
+    changedFiles.forEach(filePath => {
+      const isTestFile = this.isTestFilePath.bind(this)(filePath);
```

Comment:
```suggestion
      const isTestFile = this.isTestFilePath(filePath);
```

Reply:
Noticed that in `findRelatedTest` method of `SearchSource` will bind `this` when calling `isTestFilePath` so I did that. Not sure why we did this in previous code.